assets/js/neoforge.js

```diff
@@ -3,23 +3,39 @@ const FORGE_GAV = 'net/neoforged/neoforge'
 const LEGACY_GAV = 'net/neoforged/forge'
 const LATEST_ENDPOINT = 'https://maven.neoforged.net/api/maven/latest/version/releases/'
 const DOWNLOAD_URL = 'https://maven.neoforged.net/releases'
-//https://maven.neoforged.net/api/maven/latest/version/releases/net%2Fneoforged%2Fneoforge?filter=20.2
+// For the latest version: https://maven.neoforged.net/api/maven/latest/version/releases/net/neoforged/neoforge
+// For legacy version(s): https://maven.neoforged.net/api/maven/latest/version/releases/net/neoforged/forge
+// To filter a specific MC version: https://maven.neoforged.net/api/maven/latest/version/releases/net/neoforged/neoforge?filter=20.4
 async function loadLatestVersions(minecraftVersions) {
     for (const mcVersion of minecraftVersions) {
 	let gav;
 	let fn;
 	let mcvers;
+        let dropDown_VAL;
```

Comment:
Might want to change this for consistent indentation (you have mixed tabs and spaces, you can see it if you set github tabs to 4). But I don't mind too much, happy to merge without this change.

Reply:
Sighhh... This has been really annoying! VS Code and Github... Can you help me check again...? (pushing a commit)